file_processing/receive.py

```diff
@@ -16,12 +16,26 @@ def main():
 def process_pdf(data):
     
     database.insert_into_db(data['name'])
-   # filename = str(Path.cwd()) + '/receipts/' + data['name']
     filename = str(dirname(dirname(abspath(__file__)))) + '/receipts/' + data['name']
```

Comment:
can you define this part str(dirname(dirname(abspath(__file__)))) as global variable to use it here?